pkg/structs/flamebearer/flamebearer.go

```diff
@@ -101,24 +102,80 @@ type FlamebearerTimelineV1 struct {
 	Watermarks    map[int]int64 `json:"watermarks"`
 }
 
+type FlamebearerHeatmapV1 struct {
```

Comment:
Nit: I would not call it flamebearer heatmap — it has nothing to do with flamebearer at this point